src/model/entity/level.go

```diff
@@ -1,8 +1,98 @@
 package entity
 
+import (
+	"errors"
+	"math/rand"
+)
+
+const (
+	RoomMinWidth     = 3
+	RoomMinHeight    = 3
+	MinRoomPadding   = 1
+	numberXYSections = 3
+)
+
 type Level struct {
-	Rooms       []Room
-	Passages    []Passage
-	LevelNumber uint
-	LevelEnd    Box
+	Rooms    []Room
+	Passages []Passage
+	Number   uint
+	End      Box
+}
+
+func calculateRoomSectionSize(sizeMap Size2D[uint]) (sectionSize Size2D[uint]) {
```

Comment:
Тут не обязательно указывать тип в названии, т.к. функция небольшая, контекст узкий, из типа понятно, что это `size`. Можно просто написать:

```suggestion
func calculateRoomSectionSize(map Size2D[uint]) (section Size2D[uint]) {
```

Reply:
Я по итогу убрала из названия функции слово size, а не из аргументов, потому что в аргументах на переменную с названием map язык ругается, так как думает, что я пытаюсь карту создать (map[string]int)
<img width="586" height="74" alt="image" src="https://github.com/user-attachments/assets/0d0ef55a-881c-4495-999d-cf167e4887f9" />